pkgs/top-level/python-packages.nix

```diff
@@ -22182,6 +22182,29 @@ in modules // {
     };
   };
 
+  reno = buildPythonPackage rec {
+    name = "reno-${version}";
+    version = "1.8.0";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/r/reno/${name}.tar.gz";
+      sha256 = "1pqg0xzcilmyrrnpa87m11xwlvfc94a98s28z9cgddkhw27lg3ps";
+    };
+
+    # Tests require gnupg1 and git
+    doCheck = false;
+
+    dontStrip = true;
```

Comment:
Leaving dontStrip at true produces the following output :
```
stripping (with flags -S) in /nix/store/xs5h6gyp4fk1vl5ksx67kw9lasff1wyx-reno-1.8.0/lib  /nix/store/xs5h6gyp4fk1vl5ksx67kw9lasff1wyx-reno-1.8.0/bin 
strip:/nix/store/xs5h6gyp4fk1vl5ksx67kw9lasff1wyx-reno-1.8.0/lib/python2.7/site-packages/reno/sphinxext.pyc: File format not recognized
strip:/nix/store/xs5h6gyp4fk1vl5ksx67kw9lasff1wyx-reno-1.8.0/lib/python2.7/site-packages/reno/__init__.py: File format not recognized
strip:/nix/store/xs5h6gyp4fk1vl5ksx67kw9lasff1wyx-reno-1.8.0/lib/python2.7/site-packages/reno/__init__.pyc: File format not recognized
...
```
My understanding was that strip is only meaningful on elf binaries.